packages/fuselage/src/playground/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {
+  LiveProvider,
+  LiveEditor,
+  LiveError,
+  LivePreview,
+} from 'react-live';
+import dracula from 'prism-react-renderer/themes/dracula';
+import duotoneDark from 'prism-react-renderer/themes/duotoneDark';
+import duotoneLight from 'prism-react-renderer/themes/duotoneLight';
+import github from 'prism-react-renderer/themes/github';
+import nightOwl from 'prism-react-renderer/themes/nightOwl';
+import nightOwlLight from 'prism-react-renderer/themes/nightOwlLight';
+import oceanicNext from 'prism-react-renderer/themes/oceanicNext';
+import palenight from 'prism-react-renderer/themes/palenight';
+import shadesOfPurple from 'prism-react-renderer/themes/shadesOfPurple';
+import synthwave84 from 'prism-react-renderer/themes/synthwave84';
+import ultramin from 'prism-react-renderer/themes/ultramin';
+import vsDark from 'prism-react-renderer/themes/vsDark';
+import * as cssInJs from '@rocket.chat/css-in-js';
+import * as hooks from '@rocket.chat/fuselage-hooks';
+
+import * as components from '..';
+
+const scope = { components, React, cssInJs, hooks };
```

Comment:
Why not spread all the symbols? I don't think they collide:

```suggestion
const scope = { ...components, ...React, ...cssInJs, ...hooks };
```